src/components/cesium_map/cesium_UI.js

```diff
@@ -25,6 +25,14 @@ import { addButton } from "./elements/navigationButton";
 //  https://cesium.com/learn/ion/cesium-ion-access-tokens/
 Cesium.Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIwNzk3NjkyMy1iNGI1LTRkN2UtODRiMy04OTYwYWE0N2M3ZTkiLCJpZCI6Njk1MTcsImlhdCI6MTYzMzU0MTQ3N30.e70dpNzOCDRLDGxRguQCC-tRzGzA-23Xgno5lNgCeB4';
 
+// constant variables
+const REFRESHTIMEMS = 5000;
+const MINIMUMREFRESHDISTANCE = 50;
+const MAXIMUMREFRESHDISTANCE = 4000;
+const MAXIMUMZOOMDISTANCE = 15000000;
+const NUMBEROFPOINTS = 100000;
+const GLOBALHEADING = 90;
+const GLOBALPITCH = -90;
```

Comment:
This is great!  One small suggestion -- when you're joining words in constants it can be easier to read if you join them with underscores, e.g. `REFRESH_TIME_MS` instead of `REFRESHTIMEMS`.

Reply:
Thank you for the suggestion!